core/src/main/java/nl/nn/adapterframework/jdbc/dbms/IDbmsSupport.java

```diff
@@ -93,8 +93,9 @@
 	String getSchema(Connection conn) throws JdbcException;
 	
 	List<String> splitQuery(String query);
-	String convertQuery(Connection conn, String query, String sqlDialectFrom) throws SQLException, JdbcException;
-
+	String convertQuery(Connection conn, String query, String sqlDialectFrom, boolean updateable) throws SQLException, JdbcException;
+	// naming 'updateable' is not very clear, but it's already used. It indicates that 'updateBlob' or 'updateClob' is true.
```

Comment:
Dan zou ik er hier toch voor kiezen om er 'updateLob' van te maken.

Reply:
argument 'updateable' is komen te vervallen met introductie van QueryContext